internal/api/v1/user/merkleroots/merkleroots_api_test.go

```diff
@@ -56,3 +57,83 @@ func TestMerkleRootsAPI_MerkleRoots(t *testing.T) {
 		})
 	}
 }
+
+// Mock repository for testing
+type MockMerkleRootsRepository struct {
```

Comment:
Whats the purpose of these tests if we have them in `merkleroots_sync_test.go`

Reply:
as explained in DM, resolving it